app/src/openxr/cpp/OpenXRLayers.cpp

```diff
@@ -205,17 +205,17 @@ OpenXRLayerEquirect::IsDrawRequested() const {
 }
 
 void
-OpenXRLayerEquirect::Update(XrSpace aSpace, const XrPosef &aPose, XrSwapchain aClearSwapChain) {
+OpenXRLayerEquirect::Update(XrSpace aSpace, const XrPosef &aReorientPose, XrSwapchain aClearSwapChain) {
   OpenXRLayerPtr source = sourceLayer.lock();
   if (source) {
     swapchain = source->GetSwapChain();
   }
-  OpenXRLayerBase<VRLayerEquirectPtr, XrCompositionLayerEquirectKHR>::Update(aSpace, aPose, aClearSwapChain);
+  OpenXRLayerBase<VRLayerEquirectPtr, XrCompositionLayerEquirectKHR>::Update(aSpace, aReorientPose, aClearSwapChain);
 
   const uint numXRLayers = GetNumXRLayers();
   for (int i = 0; i < numXRLayers; ++i) {
     device::Eye eye = i == 0 ? device::Eye::Left : device::Eye::Right;
-    xrLayers[i].pose =  XrPoseIdentity();
+    xrLayers[i].pose = aReorientPose;
```

Comment:
> `pose` is an [XrPosef](https://registry.khronos.org/OpenXR/specs/1.0/man/html/XrPosef.html) defining the position and orientation of the center point of the sphere onto which the equirect image data is mapped, relative to the reference frame of the space.

See [XrCompositionLayerEquirectKHR](https://registry.khronos.org/OpenXR/specs/1.0/man/html/XrCompositionLayerEquirectKHR.html) for details.